undead-discord.css

```diff
@@ -33,6 +33,64 @@
   border-bottom: 1px solid var(--darkest-bg-color);
 }
 
+/* The following 3 blocks color the keybind-shortcut modal and buttons */
+.ddr-arrows { display: none; }
```

Comment:
If this is just a single line, might as well add it to the other `display: none;` stuff.

Reply:
Fixed. Will show up in next commit

Reply:
You don't have any stuff with `display: none;` This is the only instance of it that I can find

Reply:
https://github.com/chrisduerr/userstyles/blob/master/undead-discord.css#L140